index.adoc

```diff
@@ -1,178 +1,101 @@
 = WildFly Documentation
+:toc:
+:wildfly-latest-version: 35.0.0.Final
+:wildfly-latest-major: 35
 
-== Releases
+== WildFly Application Server
 
-|===
-|WildFly Release | Jakarta EE Version
+WildFly is a powerful, modular, & lightweight application server that helps you build amazing enterprise Java applications.
 
-|link:35[35.0.0.Final]
-|https://jakarta.ee/specifications/platform/10/apidocs/[Jakarta EE 10]
+Learn more about *link:{wildfly-latest-major}[WildFly {wildfly-latest-version}]* (or link:./archives[previous releases]) and read the link:https://www.wildfly.org/guides/[Guides].
 
+WildFly provides Jakarta EE APIs that are documented in:
 
-|===
+* https://jakarta.ee/specifications/platform/10/apidocs/[Jakarta EE 10]
+* https://jakarta.ee/specifications/platform/11/apidocs/[Jakarta EE 11]
 
-|===
-|Bootable JAR
 
-|link:bootablejar[Bootable JAR]
+== Build Applications With WildFly
 
-|===
+=== WildFly Maven Plugin
 
-|===
-|Galleon
+The WildFly Maven plugin allows you to manipulate a WildFly server directly from a Java Maven project.
+Its features include:
 
-|link:galleon[Galleon]
+* Deploy, redeploy or undeploy your application
+* Add resources
+* Execute CLI commands
+* Run a standalone server within Maven
+* Provision & package a WildFly server fit for your application
+* Build a container image with WildFly and your application
+* Build a Bootable Jar application
 
-|===
+Learn more about the link:wildfly-maven-plugin[WildFly Maven Plugin].
 
-|===
-|WildFly Galleon Plug-ins
+=== WildFly Glow
 
-|link:galleon-plugins[WildFly Galleon Plug-ins]
+WildFly Glow is a tool and a Maven plugin to automatically discover an application's requirements and provision WildFly with the right set of capabilities.
```

Comment:
I don't want to block merging on this as I think it looks really nice and much cleaner than what we have now.

The only thing I wan to know is GLOW isn't really a Maven Plugin. The wildfly-maven-plugin uses GLOW and there is a Maven Plugin for integrating with Arquillian. That said, I don't really have better wording for it :)

Reply:
@jamezp thanks for the feedback, I've amended the description. Please tell me if you're ok with it